_posts/2021-03-08-kfserving-0.5.md

```diff
@@ -29,7 +29,7 @@ KFServing 0.5 also introduces an optional model agent for request/response loggi
 
 -   [<u>TorchServe integration</u>](https://twitter.com/PyTorch/status/1346466660744511492): TorchServe now is used as implementation for KFServing PyTorch model server, it also enables model explanability with [<u>Captum</u>](https://captum.ai/), see TorchServe examples [<u>here</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/torchserve).
 
--   Triton Inference Server V2 inference REST/gRPC protocol support, see examples of serving [<u>bert</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/triton/bert) and [<u>torchscript</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/triton/torchscript) models on GPUs.
+-   Triton Inference Server V2 inference REST/gRPC protocol support, see examples of serving [<u>BERT</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/triton/bert) and [<u>torchscript</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/triton/torchscript) models on GPUs.
 
```

Comment:
missing the link for XGBoost

Reply:
can you add as suggestion to commit @yuzisun ?

Reply:
@animeshsingh added in suggestion. please help update

Reply:
done